includes/class-coblocks-form.php

```diff
@@ -453,6 +453,27 @@ public function process_form_submission( $atts ) {
 
 		}
 
+		/**
+		 * Fires before a form is submitted.
+		 *
+		 * @param array $_POST User submitted form data.
+		 * @param array $atts  Form block attributes.
+		 */
+		do_action( 'coblocks_before_form_submit', $_POST, $atts );
+
+		/**
+		 * Filter to disable the CoBlocks form emails.
+		 *
+		 * @param bool false Whether or not the emails should be disabled.
+		 */
+		$disable_emails = (string) apply_filters( 'coblocks_form_disable_emails', false );
+
+		if ( $disable_emails ) {
+
+			return true;
```

Comment:
The docblock for this function should reflect a boolean return value. And since we are returning `true` here, we should ensure a proper return value is set for the others rather than null (just to be explicit). Looks like the return of this function is used for a conditional so a boolean value makes sense.

Reply:
Good call.